cypress/support/commands/parent-commands.ts

```diff
@@ -98,7 +98,7 @@ Cypress.Commands.add('getInFixedContainer', (selector) => {
 
 Cypress.Commands.add('visualRegressionTestUrl', ({url, testName}) => {
   cy.visit(url);
-  cy.wait(1000);
+  cy.wait(2000);
```

Comment:
@SiTaggart Percy was catching regressions as a result of [R18 stricter Strict Mode](https://reactjs.org/blog/2022/03/29/react-v18.html#new-strict-mode-behaviors) which does an extra unmount/remount. We want to keep it on to catch concurrency bugs for code resiliency. As such, we need to bump the wait time (might need more) to ensure all re-rendering is done.